General_gNMI/02__Get.robot

```diff
@@ -29,6 +29,12 @@ Parameter "prefix" on root path
     When Dispatch Get Request
     Then Should received OK Response
 
+Parameter "prefix" with "path" combination
+    [Documentation]    Test the device defined nested path defined by "" variable in device YAML file.
```

Comment:
Sorry, I somehow cannot parse the sentence, too many *defined* and I keep getting lost. Also, what is *defined by "" variable*?